pkg/types/workspace.go

```diff
@@ -50,3 +50,10 @@ type WorkspaceInfo struct {
 	Projects         []*ProjectInfo `json:"projects"`
 	ProviderMetadata string         `json:"providerMetadata,omitempty"`
 } // @name WorkspaceInfo
+
+type WorkspaceState string // @name WorkspaceState
```

Comment:
I can see why you felt this block was needed. We should make proper enumeration of the container states in the future. Until then I would ask you to move this to `cmd/workspace/start.go` during this transition period since it is only used by the client and only for this autocompletion case